src/System.Management.Automation/engine/NativeCommandProcessor.cs

```diff
@@ -423,7 +400,8 @@ private void InitNativeProcess()
 
             _startPosition = new Host.Coordinates();
 
-            CalculateIORedirection(out redirectOutput, out redirectError, out redirectInput);
+            bool isWindowsApplication = IsWindowsApplication(this.Path);
```

Comment:
These names are so similar that it can be confused. Can we keep the old name for the method?

Reply:
The old name with the `Check` makes it slightly harder to understand reading the code as it's not as clear what true/false means with `Check` vs `Is`.  The alternative is to have the local variable be `isConsoleApplication` to differentiate I guess.

Reply:
We could use `bool windowsApplication` without Bulgarian prefix.
If no I agreed  with `isWindowsApplication`.

Reply:
Since this code currently is only used in one place, I think the casing diff is ok and prefer the `is` making it a bit more readable